<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>501</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/background/resource/background/bg11.jpg)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTiming">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gadugi</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;
border:2px solid rgb(255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>开始/暂停</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>605</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="callTomatoWindowButton">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:yellow;
border-radius:30px;
border:2px solid rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>番茄钟</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/picture/resource/pic/4.ico</normaloff>:/picture/resource/pic/4.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="callTaskWIndowButton">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 170, 0);
border-radius:30px;
border:2px solid rgb(255, 170, 0);
</string>
       </property>
       <property name="text">
        <string>任务</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/picture/resource/pic/8.ico</normaloff>:/picture/resource/pic/8.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="readHistoryButton">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>8</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:yellow;
border-radius:30px;
border:2px solid rgb(255, 255, 0);
</string>
       </property>
       <property name="text">
        <string>历史</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/picture/resource/pic/5.ico</normaloff>:/picture/resource/pic/5.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/picture/resource/pic/2.ico)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/icon/resource/icon/2.ico)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Timer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endTaskButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;
border:2px solid rgb(255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>结束任务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;
border:2px solid rgb(255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>归零</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
